docs/output.md

```diff
@@ -0,0 +1,23 @@
+# Output Definitions
+This page aims to explain and define the various fields that will be present in the output.
+
+## Field Definitions
+
+| Field Name | Description |
+| --- | --- |
+| **Category**| This is the type of risk found by Vault Radar. Eg. secret, PII, or NIL.
+| **Description**| This is a short human readable description or explanation of the risk.
+| **Created At**| This is the time the risk was created or introduced.
+| **Author**| This is the user associated with creating or introducing the risk.
+| **Severity**| This is a classification of the risk. **Critical** risks are things Vault Radar believe are the most deserving of user's immediate attention, followed by **High**, **Medium**, and **Info**.
+| **Is Historic**| This means the risk was first created or introduced in version that precedes the most recent version.
```

Comment:
It also means that the secret is NOT present in the most recent version of content.